vue3_ts/src/core/enums/index.ts

```diff
@@ -0,0 +1 @@
+export * from '@/views/veevalidate-yup/veevalidate-yup.enums'
```

Comment:
@v-romanyuk should we have additional folder: src/core/enums? I think this can be placed inside: @/src/types/index.ts?

Reply:
No, we use only .types.ts extension to gather all ts definitions like types, interfaces, enums and so forth. So in this case I would place it into @/views/veevalidate-yup/veevalidate-yup.types.ts' and therefore create an enum in that file.